src/cmds/net/arping.c

```diff
@@ -114,15 +113,47 @@ int main(int argc, char **argv) {
 	from.s_addr = in_dev->ifa_address;
 	strncpy(from_b, inet_ntoa(from), sizeof(from_b) - 1);
 	printf("ARPING %s from %s %s\n", dst_b, from_b, in_dev->dev->name);
+
+	struct timeval t1, t2;
+	int ret, microseconds, milliseconds;
```

Comment:
Please follow our code style - https://github.com/embox/embox/wiki/Code-Style, we declare variables at the start of a function (or blocks like `for`, `while`).